_posts/2021-11-11-java-release-notes.md

```diff
@@ -310,51 +288,33 @@ try(BufferedReader reader = br3){
 }
 ```
 Inside `try` definition, we assigned our previously created reader to the new variable. Now we know that our reader will get closed every time.
-#### Improvement upon Try-with-resource
-How can we improve [the example](#improvement-on-autoclosable) from above? 
 
-Well, defining a variable in one place only to reassign it to another inside `try` definition is a little bit of pain in the eye.
-With Java 9, we got changes with which we can avoid that scenario. We will be looking into the same example of reading input string through `BufferedReader`:
-```java
-final BufferedReader br2 = new BufferedReader(new StringReader("Hello world example2!"));
-try(br2){
-    System.out.println(br2.readLine());
-}
-catch (IOException e){
-    System.out.println("Error happened!");
-}
-```
-Now we don't need to create a new variable only to autoclose it inside the `try-catch` block. We are using the same one from the original definition.
-
-To read more about `try-with-resources` feature please refer to [oracle page.](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)
+To read more about the try-with-resources feature please refer to [the oracle page.](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)
 ### Diamond Syntax with Inner Anonymous Classes
 #### What Java 9 Fixed
-Before Java 9 we couldn't use a diamond operator inside inner anonymous class.
+Before Java 9 we couldn't use a diamond operator inside the inner anonymous class.
 
 For our example, we will create an abstract class `AppendingString`. It has only one method that appends two strings with `-` between them. 
 Since this is an abstract class, we will use the anonymous class for providing the implementation for the `append()` method:
 ```java
-public abstract static class AppendingString<T>{
-  public abstract T append(String a, String b);
+public abstract static class StringAppender<T>{
+    public abstract T append(String a, String b);
 }
 
 public static void main(String[] args) {
-    AppendingString<String> appending = new AppendingString<>() {
+    StringAppender<String> appending = new StringAppender<String>() {
         @Override
         public String append(String a, String b) {
             return new StringBuilder(a).append("-").append(b).toString();
         }
     };
-
-    String result = appending.append("Reflectoring", "Blog");
-    System.out.println(result);
 }
 ```
 We are using the diamond operator to tell our method which type we expect.
```

Comment:
TODO: After your change I can't find a diamond operator in the code example at all anymore?!? Also, please add the `DiamondOperator` class around the code example so that people can make a connection to the error message.